plugin/evm/atomic/txpool/mempool.go

```diff
@@ -195,13 +196,13 @@ func (m *Mempool) ForceAddTx(tx *Tx) error {
 // checkConflictTx checks for any transactions in the mempool that spend the same input UTXOs as [tx].
 // If any conflicts are present, it returns the highest gas price of any conflicting transaction, the
 // txID of the corresponding tx and the full list of transactions that conflict with [tx].
-func (m *Mempool) checkConflictTx(tx *Tx) (uint64, ids.ID, []*Tx, error) {
+func (m *Mempool) checkConflictTx(tx *atomic.Tx) (uint64, ids.ID, []*atomic.Tx, error) {
 	utxoSet := tx.InputUTXOs()
 
 	var (
-		highestGasPrice             uint64 = 0
-		conflictingTxs              []*Tx  = make([]*Tx, 0)
-		highestGasPriceConflictTxID ids.ID = ids.ID{}
+		highestGasPrice             uint64       = 0
+		conflictingTxs              []*atomic.Tx = make([]*atomic.Tx, 0)
+		highestGasPriceConflictTxID ids.ID       = ids.ID{}
```

Comment:
Not super relevant.. But I thought this was weird.
```suggestion
		highestGasPrice             uint64
		conflictingTxs              []*atomic.Tx
		highestGasPriceConflictTxID ids.ID
```